frappe/translate.py

```diff
@@ -113,7 +113,14 @@ def get_dict(fortype, name=None):
 		elif fortype=="jsfile":
 			messages = get_messages_from_file(name)
 		elif fortype=="boot":
-			messages = get_messages_from_include_files()
+			messages = []
+			apps = frappe.get_all_apps(True)
+			for app in apps:
+				messages.extend(get_server_messages(app))
+			messages = deduplicate_messages(messages)
+
+			messages += get_messages_from_include_files()
+			messages += frappe.db.sql("""select "navbar", item_label from `tabNavbar Item` where item_label is not null""")
```

Comment:
Better to add a hook that defines Doctypes and their fields that will be translatable.

Reply:
here navibar is boot level same as all doctypes and modules, so I do suggest to include it in the boot messages list.